r-package/scripts/create_model_output.R

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env Rscript
+root_dir <- here::here()
```

Comment:
So I think this is very similar to `run_model.R` script in `./r-package/scripts/run_model.R` can we consolidate these.

In fact I think that script will be broken since last PR as it was using standalone model data. 

Good to go with this, but can we add the `usage` here with docopt and let the caller pass in output dir.